### PR TITLE
[i18n-zh] Translate add_new_pipeline.md into Chinese

### PR DESCRIPTION
**Pull request by windsonsea**
Add zh text to this page:

```
docs/source/zh/add_new_pipeline.md
```
cc @stevhliu @Fan-Lin @jiaqiw09

### CODE REVIEW

File: docs/source/zh/add_new_pipeline.md [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+
+⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+-->
+
+# 如何创建自定义流水线？
+
+在本指南中，我们将演示如何创建一个自定义流水线并在 [Hub](https://hf.co/models) 进行分享，或将其添加到 🤗 Transformers 库中。
```
Comment (Fan-Lin) [inline docs/source/zh/add_new_pipeline.md:18]:
```suggestion
在本指南中，我们将演示如何创建一个自定义流水线并分享到 [Hub](https://hf.co/models)，或将其添加到 🤗 Transformers 库中。
```

File: docs/source/zh/add_new_pipeline.md [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+
+首先，你需要决定流水线将能够接受的原始条目。它可以是字符串、原始字节、字典或任何看起来最可能是期望的输入。
+尽量保持这些输入尽可能纯粹的 Python，因为这样可以更容易地实现兼容性（甚至通过 JSON 在其他语言之间）。
```
Comment (Fan-Lin) [inline docs/source/zh/add_new_pipeline.md:21]:
```suggestion
尽量保持输入为纯 Python 语言，因为这样可以更容易地实现兼容性（甚至通过 JSON 在其他语言之间）。
```

File: docs/source/zh/add_new_pipeline.md [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+        outputs = self.model(**model_inputs)
+        # Maybe {"logits": Tensor(...)}
+        return outputs
+
+    def postprocess(self, model_outputs):
+        best_class = model_outputs["logits"].softmax(-1)
+        return best_class
+```
+
+这种分解的结构是为了支持相对无缝地在 CPU/GPU 上进行支持，同时支持在不同线程上在 CPU 上进行预处理/后处理。
```
Comment (Fan-Lin) [inline docs/source/zh/add_new_pipeline.md:54]:
```suggestion
这种分解的结构旨在为 CPU/GPU 提供相对无缝的支持，同时支持在不同线程上对 CPU 进行预处理/后处理。
```

File: docs/source/zh/add_new_pipeline.md [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+    if "maybe_arg" in kwargs:
+        preprocess_kwargs["maybe_arg"] = kwargs["maybe_arg"]
+
+    postprocess_kwargs = {}
+    if "top_k" in kwargs:
+        postprocess_kwargs["top_k"] = kwargs["top_k"]
+    return preprocess_kwargs, {}, postprocess_kwargs
+```
+
+尽量保持输入/输出非常简单，最好是可 JSON 序列化的，因为这样可以使流水线的使用非常简单，而不需要用户了解新的对象类型。
```
Comment (Fan-Lin) [inline docs/source/zh/add_new_pipeline.md:102]:
```suggestion
尽量保持简单输入/输出，最好是可 JSON 序列化的，因为这样可以使流水线的使用非常简单，而不需要用户了解新的对象类型。
```

File: docs/source/zh/add_new_pipeline.md [hunk 5]
```diff
@@ -0,0 +1,238 @@
...
+    pt_model=AutoModelForSequenceClassification,
+    default={"pt": ("user/awesome_model", "abcdef")},
+    type="text",  # current support type: text, audio, image, multimodal
+)
+```
+
+## 在 Hub 上分享你的流水线
+
+要在 Hub 上分享你的自定义流水线，你只需要将 `Pipeline` 子类的自定义代码保存在一个 Python 文件中。
+例如，假设我们想要为句对分类使用一个自定义流水线，如下所示：
```
Comment (Fan-Lin) [inline docs/source/zh/add_new_pipeline.md:134]:
```suggestion
例如，假设我们想使用一个自定义流水线进行句对分类，如下所示：
```

File: docs/source/zh/add_new_pipeline.md [hunk 6]
```diff
@@ -0,0 +1,238 @@
...
+        probabilities = softmax(logits)
+
+        best_class = np.argmax(probabilities)
+        label = self.model.config.id2label[best_class]
+        score = probabilities[best_class].item()
+        logits = logits.tolist()
+        return {"label": label, "score": score, "logits": logits}
+```
+
+这个实现是框架无关的，适用于 PyTorch 和 TensorFlow 模型。如果我们将其保存在一个名为
```
Comment (Fan-Lin) [inline docs/source/zh/add_new_pipeline.md:172]:
```suggestion
这个实现与框架无关，适用于 PyTorch 和 TensorFlow 模型。如果我们将其保存在一个名为
```

**Review by stevhliu — APPROVED** [review]
Format and structure looks good to me. Happy to merge once you've incorporated @Fan-Lin 's feedback 😄

**Comment by windsonsea** [comment]
Thanks for your review. All suggestions are applied.

**Comment by windsonsea** [comment]
Hello @stevhliu, I am interested in becoming a member of Hugging Face. Could you please provide information on how to apply and where I can raise any issues or submit a pull request? Thank you.

**Review by Fan-Lin — APPROVED** [review]

**Comment by stevhliu** [comment]
Thanks for your interest! Feel free to check out the job postings [here](https://apply.workable.com/huggingface/?lng=en) and apply 🤗
